pkgs/by-name/mi/mingtest/package.nix

```diff
@@ -0,0 +1,25 @@
+{
+  stdenv,
+  cmake,
+  texliveFull
+  }:
+
+  stdenv.mkDerivation {
+     name = "mingtest";
+     version = "0.1.9-1";
+     src = fetchGit {
+        url = "https://github.com/craflin/mingtest.git";
+        ref = "master";
+  };
```

Comment:
You are init this package in specify version, thus you have to 
use something like follows OR just change to fetchFromGitHub to use it's version param.

```
builtins.fetchGit {
  url = "https://github.com/craflin/mingtest.gi";
  ref = "refs/tags/0.1.9-1";
}
```